lib/rialto/etl/configs/stanford_organizations_to_sparql_statements.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'traject_plus'
+require 'rialto/etl/readers/stanford_organizations_json_reader'
+require 'rialto/etl/writers/sparql_statement_writer'
+require 'rialto/etl/namespaces'
+
+extend TrajectPlus::Macros
+extend TrajectPlus::Macros::JSON
+extend Rialto::Etl::NamedGraphs
+extend Rialto::Etl::Vocabs
+
+settings do
+  provide 'writer_class_name', 'Rialto::Etl::Writers::SparqlStatementWriter'
+  provide 'reader_class_name', 'Rialto::Etl::Readers::StanfordOrganizationsJsonReader'
+end
+
+# The named graph to place these triples into.
+to_field '@graph', literal(STANFORD_ORGANIZATIONS_GRAPH.to_s), single: true
+
+# Subject
+to_field '@id', extract_json('$.alias'), single: true
```

Comment:
I'm wondering why we need two `@id*` fields?  Couldn't you just construct the one here?  If not, can you add a comment about why not?

Reply:
Sure. We used the id (profileId) to construct other URIs by the SPARQL writer. For example, for the person `http://sul.stanford.edu/rialto/agents/people/1234`, the URI for his name vcard is `http://sul.stanford.edu/rialto/context/names/1234`.